src/bindist/autoemu.bat

```diff
@@ -1,8 +1,10 @@
 @echo off
 rem generic autoexec.bat for DOSEMU + any DOS
 rem must be edited in most circumstances
-path c:\dos;c:\windows\command;e:\dosemu
-lredir d: linux\fs\${DOSEMU_LIB_DIR}/freedos ro
+set PATH=c:\windows\command;e:\dosemu

```

Comment:
Yes, with 'system -s' you can pass anything
you want. IIRC for ms-dos >=7.x you are going
to pass c:\windows\command instead of c:\dos,
so you'll have to use system -s.